src/fauxmoESP.h

```diff
@@ -80,7 +80,7 @@ typedef struct {
     char * name;
     bool state;
     unsigned char value;
-    char uniqueid[28];
```

Comment:
Note that this char buffer now has length 27 instead of 28. I *think* 28 was a bug, but I'm admittedly not much of a C programmer, so I could be wrong. The default unique ID that you're generating is 26 actual characters long, plus 1 more for the null terminator, giving 27, so I've converged everything to using `FAUXMO_DEVICE_UNIQUE_ID_LENGTH` which is defined as 27.